paddle/fluid/framework/operator.cc

```
@@ -90,8 +110,10 @@ static int GetRowSize(const Scope& scope, const std::string& name) {
     return -1;
   }
 
-  if (var->IsType<SelectedRows>()) {
-    return var->Get<SelectedRows>().rows().size();
+  if (var->IsInitialized()) {
```

I think we should no have the condition that var is not initialized. If so, just let it crash. Tensor not initialized can happen.

---

OK, I will remove this condition later